setup.cfg

```diff
@@ -23,7 +23,6 @@ classifiers =
     Operating System :: OS Independent
     Programming Language :: Python
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.6
```

Comment:
And a note somewhere that the next release will drop 3.6 (if it wasn't dropped already)